test/integration_test.rb

```diff
@@ -387,7 +387,7 @@ def assert_telemetry(request, expected_uri = @uri.path.sub(Dir.home, "~"))
     assert_equal(expected_uri, telemetry_response.dig(:params, :uri))
     assert_equal(RubyLsp::VERSION, telemetry_response.dig(:params, :lspVersion))
     assert_equal(request, telemetry_response.dig(:params, :request))
-    assert_in_delta(0.5, telemetry_response.dig(:params, :requestTime), 2)
+    assert_in_delta(0.5, telemetry_response.dig(:params, :requestTime), ENV["CI"] ? 5 : 2)
```

Comment:
This is a strange usage of delta? We expect the query to be processed in 0.5s +/- 5s?

Reply:
That's a good point. The idea was just to check that this was present and that the value made sense, but with all of the CI variation this assertion is making basically no sense.

What do people think about changing this to be
```ruby
assert_instance_of(Float, telemetry_response.dig(:params, :requestTime))
```
Just to check that there's some float value in there.

Reply:
Yeah, I think the `assert_instance_of` will show the intent better 👍